### PR TITLE
test(dm): show full result of ps on kill fail

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #5089

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: dm/tests/_utils/wait_process_exit [hunk 1]
```diff
@@ -16,5 +16,5 @@ while [ $WAIT_COUNT -lt 120 ]; do
 	((WAIT_COUNT++))
 done
 
-echo "process $process didn't exit after 120 seconds, current processlist: $(pgrep $process)"
+echo "process $process didn't exit after 120 seconds, current processlist: $(ps -ef | grep $process | grep -v 'grep')"
```
Comment (lance6716) [inline dm/tests/_utils/wait_process_exit:19]: maybe `ps $process` is enough

**Review by lance6716 — APPROVED** [review]

File: dm/tests/_utils/wait_process_exit (hunk 1 above)
Comment (D3Hunter) [reply dm/tests/_utils/wait_process_exit:19]: > maybe `ps $process` is enough.

`ps` can't filter by name, has to grep

**Review by buchuitoudegou — APPROVED** [review]
lgtm. It seems this PR isn't about to resolve the issue but rather probe the root cause when it happens again🤣

**Pull request merged**